Platform/Apple/virtual/src/font/fontEngine.s

```diff
@@ -496,16 +487,14 @@ Adv210	STA CursColL	;position to 154
 	STA WrdWdth	;and, clear Word Width total
 	STA TtlScrl	;and ticker scroll total
 	LDA CursRow	;Get vertical {0..191}
-	CLC
```

Comment:
>> @@ -496,16 +487,14 @@ Adv210	STA CursColL	;position to 154
>>  	STA WrdWdth	;and, clear Word Width total
>>  	STA TtlScrl	;and ticker scroll total
>>  	LDA CursRow	;Get vertical {0..191}
>> -	CLC
>
> This proposed change seems to assume that carry is being cleared earlier.
> Please correct me if I'm wrong, but I think that since at this point the
> prior BMI failed, the carry has actually been set, and it is indeed
> necessary to clear it here.

I'm wrong about this one.  There's another code path which requires
it, and which I overlooked.  I will correct the submission in a
moment.